experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -512,11 +517,97 @@ export interface Sequence<TTypes extends AllowedTypes> extends TreeField {
 
 	readonly length: number;
 
-	// TODO: more and/or better editing APIs. As is, this can't express moves.
-	replaceRange(
+	/**
+	 * Inserts new item(s) at a specified location.
+	 * @param index - The index at which to insert `value`.
+	 * @param value - The content to insert.
+	 */
+	insertAt(index: number, value: FlexibleNodeContent<TTypes>[]): void;
```

Comment:
What are the merge semantics here? I think it would be valuable to call them out. Are we inserting after the previous item? Before the following item?

Reply:
Neither; all methods shipped as part of MVP are index-centric, so they will result in insertion at the index specified unless concurrent changes push them forward (first-write-wins). When we introduce anchored changes, we can add those docs, but for now it's intuitive enough to omit (I validated this opinion with Noah/Alex).

Reply:
Okay. That's probably fair. Still might be worth noting that. If I were looking at this API (as I was as a reviewer), that is one of the first questions I would be asking.